compiler/src/iree/compiler/Dialect/Flow/Transforms/RegionOpUtils.cpp

```diff
@@ -43,6 +43,12 @@ static llvm::cl::opt<int> clInlineConstantByteLength(
                    "into a dispatch region or 0 to disable inlining."),
     llvm::cl::init(256));
 
+// TODO(#18457, #18447): Remove once backends support gather fusion.
+static llvm::cl::opt<bool>
```

Comment:
nit : `s/iree-dispatch-creation/iree-flow`.

Maybe some additional cleanup to move this method out of Flow.